src/utils/borrowPools/mainnetPools.js

```diff
@@ -2037,7 +2037,7 @@ export default [
     ltv: 98,
     borrowFee: 0,
     isSwappersActive: false,
-    is0xSwap: true,
+    is0xSwap: false,
```

Comment:
if we want to use the 0x swapper, just tell me and I'm going to deploy a new one that refunds if case of left over

Reply:
I do not know why the guys changed the parameter here - it does not affect anything as we did not add swappers here